RDKShell/CHANGELOG.md

```diff
@@ -16,6 +16,11 @@ All notable changes to this RDK Service will be documented in this file.
 
 * For more details, refer to [versioning](https://github.com/rdkcentral/rdkservices#versioning) section under Main README.
 
+## [1.2.3] - 2022-09-08
```

Comment:
Pls update minor version and reset patch version. So should move to 1.3.0

Reply:
Also need to update the macros in RDKShell.cpp
#define API_VERSION_NUMBER_MAJOR 1
#define API_VERSION_NUMBER_MINOR 2
#define API_VERSION_NUMBER_PATCH 2